docs/Changelog.md

```diff
@@ -28976,6 +28976,71 @@ This version of the operator has been available since version 23 of the default
 <dd>The type of the input `y_zero_point` and the output `y`.</dd>
 </dl>
 
+### <a name="RMSNormalization-23"></a>**RMSNormalization-23**</a>
+
+  This is RMS normalization defined in ONNX as function as described in the paper https://arxiv.org/pdf/1910.07467.
+        The overall computation can be split into two stages. The root mean squared norm is taken over the last D dimensions,
```

Comment:
In stage 1, I think we also need a stash_type attribute to control the precision of the accumulation. See LayerNorm and GroupNorm.